Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -1,37 +1,224 @@
-﻿namespace BootCamp.Chapter
+﻿using System;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+
+namespace BootCamp.Chapter
 {
     public static class BalanceStats
     {
         /// <summary>
         /// Return name and balance(current) of person who had the biggest historic balance.
-        /// </summary>
-        public static string FindHighestBalanceEver(string[] peopleAndBalances)
+        /// </summary>       
+        public static string FindHighestBalanceEver(string[] peoplesBalances)
         {
-            return "";
+            if (IsNullOrEmpty(peoplesBalances))
+
+                return "N/A.";
+
+            var (peopleNames, cash) = CalculatePB(peoplesBalances, "max");
```

Comment:
I don't know what PB is.

Reply:
CalculatePeoplesBalances takes a bit more memory but I think it's required. You're totally right, who will understand my code.